mimesis/data/gr/address.json

```diff
@@ -0,0 +1,399 @@
+{
+  "address_fmt": "{st_sfx} {st_num} {st_name}",
+  "city": [
+	"Άγιος Νικόλαος",
+	"Άργος",
+	"Άρτα",
+	"Έδεσσα",
+	"Αίγιο",
+	"Αγρίνιο",
+	"Αθήνα",
+	"Αλεξάνδρεια",
+	"Αλεξανδρούπολη",
+	"Αμαλιάδα",
+	"Βάρη",
+	"Βέροια",
+	"Βόλος",
+	"Γάζι",
+	"Γιαννιτσά",
+	"Γλυκά Νερά",
+	"Γρεβενά",
+	"Δράμα",
+	"Ελευσίνα",
+	"Ερμούπολη",
+	"Ηράκλειο",
+	"Θέρμη",
+	"Θήβα",
+	"Θεσσαλονίκη",
+	"Ιαλυσός",
+	"Ιεράπετρα",
+	"Ιωάννινα",
+	"Κέρκυρα",
+	"Καβάλα",
+	"Καλαμάτα",
+	"Καρδίτσα",
+	"Καστοριά",
+	"Κατερίνη",
+	"Κιλκίς",
+	"Κοζάνη",
+	"Κομοτηνή",
+	"Κορωπί",
+	"Κως",
+	"Κόρινθος",
+	"Λάρισα",
+	"Λαμία",
+	"Λιβαδειά",
+	"Λουτράκι",
+	"Μάνδρα",
+	"Μέγαρα",
+	"Μεσολόγγι",
+	"Μυτιλήνη",
+	"Νάουσα",
+	"Νέα Μάκρη",
+	"Ναύπακτος",
+	"Ναύπλιο",
+	"Ξάνθη",
+	"Ορεστιάδα",
+	"Πάτρα",
+	"Παιανία",
+	"Παλλήνη",
+	"Πειραιάς",
+	"Περαία",
+	"Πρέβεζα",
+	"Πτολεμαΐδα",
+	"Πύργος",
+	"Ρέθυμνο",
+	"Ραφήνα",
+	"Ρόδος",
+	"Σέρρες",
+	"Σαλαμίνα",
+	"Σπάρτη",
+	"Τρίκαλα",
+	"Τρίπολη",
+	"Τύρναβος",
+	"Φλώρινα",
+	"Χίος",
+	"Χαλκίδα",
+	"Χανιά",
+	"Ωραιόκαστρο"
+  ],
+  "country": {
+	"name": [
+		"Biélorussie",
```

Comment:
Extra indentation. Please, format all json data which you add using [jsoneditoronline](http://jsoneditoronline.org/)

Reply:
I tried to use the service and the json either remained as-is or didn't format correctly. Check the [json](https://imgur.com/ThCHtSn) after i saved to file. Not sure why it didn't work

Reply:
@TsimpDim Look here: http://jsoneditoronline.org/?id=f7c9d1024ce7ad7f246c8dffd337ff1d.

Insert json data click on `>` and `<`.

Reply:
@TsimpDim Also it would be awesome, if you will format data in files by alphabet. You can do it using [onlinejson editor]((http://i.imgur.com/mRqfgiE.png).)